BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -223,6 +223,11 @@ - (void) autoIdentifyCheck {
 
 ///Update current UNAuthorizationStatus in BlueshiftAppData on app launch and on app didBecomeActive
 - (void)checkUNAuthorizationStatus {
+    //Skipping push authorization change check when tracking is disabled
+    //to not miss the automatic identify event
+    if ([BlueShift sharedInstance].isTrackingEnabled == NO) {
```

Comment:
we are calling this method after this flag check, right?

Reply:
Yes, but this method is called from other places like SDK initialization or push permission code. If the condition is not added, the SDK will detect the push permission is disabled, and it will try to fire the auto identify event when SDK tracking is disabled. As tracking is disabled, We will lose this auto identify in this case.
Just to not lose the auto-identify event, added this check here.